doc/api/http2.md

```diff
@@ -1473,6 +1474,8 @@ added: v8.4.0
     sending a `PRIORITY` frame to the connected peer.
 
 Updates the priority for this `Http2Stream` instance.
+The support for priority weights has been removed in [latest versions of nghttp2 (1.65.0)][nghttp2-priority],
```

Comment:
I think we should move this description to the YAML `changes:` as this will be out-of-date as soon as nghttp releases more versions.